### DISCUSSION TITLE
Phishing Page

### DISCUSSION THREAD

**Discussion by milannshrestha** [vercel/community#6727] [Help] [answered]
### Question

Impersonating Pancakeswap: 
hxxps://pancakev2-drirxno8g-brilliantedgar [.] vercel.app/

**Comment by milannshrestha** [comment]
pancakeswapclaim6754 [.] vercel.app/

**Reply by milannshrestha** [reply]
00237362 
 
 cc @amyegan

**Reply by amyegan** [reply, marked as answer]
Thanks for letting us know about a site you believe is dangerous. You've already done the right thing by sending your report to [abuse@vercel.com](mailto:abuse@vercel.com). Please follow up with your open case via email if you have any questions or additional information to share.

Please note that this community is not the appropriate place to share these reports as I and the other members of this community do not have access to the systems and tools needed to handle them. 

Thank you again for reporting potentially harmful sites and for your patience while reports are investigated.

**Reply by milannshrestha** [reply]
bantuantiktoksellercenter.vercel.app - 00244512

**Reply by milannshrestha** [reply]
@amyegan

**Reply by amyegan** [reply]
Thanks again for emailing these reports. This community is not the appropriate place to share them as no one in the community forum is equipped to investigate these reports. 

Vercel has a team dedicated to investigating abuse reports. These reports are assigned to them and will be addressed as quickly as possible. We appreciate your patience while a long list of reports are being investigated.

[a comment by an automated account is not shown]
